controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -409,7 +409,29 @@ func (in *KubeadmControlPlane) validateVersion(previousVersion string) (allErrs
 		return allErrs
 	}
 
-	// since upgrades to the next minor version are allowed, irrespective of the patch version
+	// Check if we're trying to upgrade to Kubernetes v1.19.0, which is not supported.
+	//
+	// In Kubernetes v1.19.0, the certificates API (which does CertificateSigningRequests
+	// for node bootstrapping, among other things) introduced version v1.
+	// The kube-controller-manager switched to only speaking v1 of this API.
+	//
+	// If you have a control plane with a load balancer in front, it is possible/likely that during an upgrade from 1.18 to 1.19,
+	// the 1.18 kube-controller-manager will be removed (as part of deleting one of the old machines),
+	// and the 1.19 kube-controller-manager will potentially acquire the leader election lock.
+	// It then connects to the load balancer, and there’s a chance it gets routed to one of the remaining 1.18 apiservers.
+	// If/when this happens, the kube-controller-manager will be unable to process any new CertificateSigningRequests
+	// (because it’s asking for certificates v1, which doesn’t exist in 1.18). This means all new node bootstrapping will fail.
+	if fromVersion.NE(toVersion) && toVersion.Equals(semver.MustParse("1.19.0")) {
+		allErrs = append(allErrs,
+			field.Forbidden(
+				field.NewPath("spec", "version"),
+				fmt.Sprintf("cannot update Kubernetes version to v1.19.0"),
```

Comment:
Can we make the error message more verbose by linking to an issue? users who run into this error might be confused as to why that is and won't necessarily be reading the code comments

Reply:
Done ✔️